### DISCUSSION TITLE
When you support Collection|Array field type?

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#53] [Q&A] [answered]
I need create attributes with Collection|Array field type currently. But platform core doesn't support this field type. When you plan add this field type?

Or can I somehow use arrays in product attributes?

**Comment by alecritson** [comment]
When you say arrays, do you mean you need a select/dropdown field? Or is this something where you can add multiple values and it will return an array of the values you added?

**Reply by devmansurov** [reply]
Add multiple values and get added values from attribute. For example multiple image links from CDN

**Reply by glennjacobs** [reply]
Like a list of values I guess. So you can keep adding more.

**Reply by devmansurov** [reply]
Yes but it's must be dynamic list of values instead pre-defined list of values. I can add more attributes as Text field but it's bad idea

**Comment by glennjacobs** [comment, marked as answer]
Soon 😎

![image](https://user-images.githubusercontent.com/647407/149493984-5157fbc5-c941-4b35-9a18-4b4e7c9c42de.png)

**Reply by devmansurov** [reply]
Great) I'm waiting for the release with this update

**Reply by devmansurov** [reply]
> Soon 😎
> 
> ![image](https://user-images.githubusercontent.com/647407/149493984-5157fbc5-c941-4b35-9a18-4b4e7c9c42de.png)

And it would also be nice if you provide a field to choose the type to fill. For example field type text, image, richtext e.t.c

**Reply by glennjacobs** [reply]
That's certainly a good idea, but more advanced. Might make a good addon.
